redbot/cogs/streams/streamtypes.py

```diff
@@ -81,6 +82,7 @@ def __repr__(self):
 class YoutubeStream(Stream):
 
     token_name = "youtube"
+    platform_name = "Youtube"
```

Comment:
YouTube uses a different casing for its name:
```suggestion
    platform_name = "YouTube"
```